Makefile

```diff
@@ -36,7 +36,7 @@ clean-simulator:
 	./clean-containers.sh camera-simulator
 
 build-profile-launcher:
-	@cd ./configs/opencv-ovms/cmd_client && $(MAKE) build
+	@cd ./configs/opencv-ovms/cmd_client && docker build --target bin --output=${PWD}/configs/opencv-ovms/cmd_client .
```

Comment:
we can update makebuild target in the other side and just call make build here the same way. i.e: refactor the make build target in another makefile so they are all encapsulated in that place

Reply:
this one i meant: https://github.com/intel-retail/automated-self-checkout/blob/main/configs/opencv-ovms/cmd_client/Makefile